README.md

```diff
@@ -1,137 +1,338 @@
+
+### Agent with Plugin - .NET
+
+```csharp
+using System.ComponentModel;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+var builder = Kernel.CreateBuilder();
+builder.AddAzureOpenAIChatCompletion(
+                Environment.GetEnvironmentVariable("AZURE_OPENAI_DEPLOYMENT"),
+                Environment.GetEnvironmentVariable("AZURE_OPENAI_ENDPOINT"),
+                Environment.GetEnvironmentVariable("AZURE_OPENAI_API_KEY")
+                );
+var kernel = builder.Build();
+
+kernel.Plugins.Add(KernelPluginFactory.CreateFromType<MenuPlugin>());
+
+ChatCompletionAgent agent =
+    new()
+    {
+        Name = "SK-Assistant",
+        Instructions = "You are a helpful assistant.",
+        Kernel = kernel,
+        Arguments = new KernelArguments(new PromptExecutionSettings() { FunctionChoiceBehavior = FunctionChoiceBehavior.Auto() })
+
+    };
+
+ChatMessageContent message = new(AuthorRole.User, "What is the price of the soup special?");
+
+await foreach (AgentResponseItem<ChatMessageContent> response in agent.InvokeAsync(message))
+{
+    Console.WriteLine(response.Message);
+    // The price of the Clam Chowder, which is the soup special, is $9.99.
+}
+
+sealed class MenuPlugin
+{
+    [KernelFunction, Description("Provides a list of specials from the menu.")]
+    public string GetSpecials() =>
+        """
+        Special Soup: Clam Chowder
+        Special Salad: Cobb Salad
+        Special Drink: Chai Tea
+        """;
+
+    [KernelFunction, Description("Provides the price of the requested menu item.")]
+    public string GetItemPrice(
+        [Description("The name of the menu item.")]
+        string menuItem) =>
+        "$9.99";
+}
+```
+
+### Multi-Agent System - Python
+
+Build a system of specialized agents that can collaborate:
+
+```python
+import asyncio
+from semantic_kernel.agents import ChatCompletionAgent
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+from semantic_kernel.agents import ChatHistoryAgentThread
+from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion
+
+billing_agent = ChatCompletionAgent(
+    service=AzureChatCompletion(), 
+    name="BillingAgent", 
+    instructions="You handle billing issues like charges, payment methods, cycles, fees, discrepancies, and payment failures."
+)
+
+refund_agent = ChatCompletionAgent(
+    service=AzureChatCompletion(),
+    name="RefundAgent",
+    instructions="Assist users with refund inquiries, including eligibility, policies, processing, and status updates.",
+)
+
+triage_agent = ChatCompletionAgent(
+    service=OpenAIChatCompletion(),
+    name="TriageAgent",
+    instructions="Evaluate user requests and forward them to BillingAgent or RefundAgent for targeted assistance."
+    " Provide the full answer to the user containing any information from the agents",
+    plugins=[billing_agent, refund_agent],
+)
+
+thread: ChatHistoryAgentThread = None
```

Comment:
```suggestion
thread = None
```
We can simplify the imports by removing the type, no worries that we're not explicit here.